packages/http/package.json

```diff
@@ -16,13 +16,14 @@
     "node": ">=8"
   },
   "dependencies": {
-    "@stoplight/json-schema-faker": "^0.5.0-rc18",
     "@stoplight/prism-core": "^3.0.0-alpha.14",
     "accepts": "^1.3.7",
     "ajv": "^6.0.0",
     "ajv-oai": "^1.0.0",
     "axios": "^0.18.0",
     "caseless": "^0.12.0",
+    "@stoplight/json-schema-faker": "",
```

Comment:
We had to bring this chap back?

Reply:
Unfortunately yes; it's still because of the damn JSONpath library that it's used internally. I've tried to get rid of it but I decided to give up…for now. The feature is more important right now.